shared/chat/manage-channels/index.desktop.js

```diff
@@ -108,13 +108,27 @@ const ManageChannels = (props: Props) => {
   }
   return (
     <PopupDialog onClose={props.onClose} styleCover={_styleCover} styleContainer={_styleContainer}>
+      {props.canCreateChannels && (
```

Comment:
I also moved the "New chat channel" button outside of the flexbox – this doesn't change anything since absolute positioning invalidates being a flex child and I think better reflects the layout of content on the page.